echopype/echodata/convention/1.0.yml

```diff
@@ -57,6 +57,8 @@ variable_and_varattributes:
       standard_name: sound_frequency
```

Comment:
frequency should be removed, right?

Reply:
@emiliom Yes, since this is under `beam_coord_default` I think we should remove it. I just looked through the code and it seems like removing it should not cause any issues. If you want me to, I will go ahead and remove it and rerun the full test suite to make sure there are no problems.

Reply:
Yeah, let's remove it